shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-api/src/main/java/org/apache/shardingsphere/encrypt/api/config/rule/EncryptColumnRuleConfiguration.java

```diff
@@ -18,22 +18,45 @@
 package org.apache.shardingsphere.encrypt.api.config.rule;
 
 import lombok.Getter;
-import lombok.RequiredArgsConstructor;
 
 /**
  * Encrypt column rule configuration.
  */
-@RequiredArgsConstructor
 @Getter
 public final class EncryptColumnRuleConfiguration {
     
     private final String logicColumn;
     
+    private final String logicDataType;
+    
     private final String cipherColumn;
     
+    private final String cipherDataType;
+    
     private final String assistedQueryColumn;
     
+    private final String assistedQueryDataType;
+    
     private final String plainColumn;
     
+    private final String plainDataType;
+    
     private final String encryptorName;
+    
+    public EncryptColumnRuleConfiguration(final String logicColumn, final String cipherColumn, final String assistedQueryColumn, final String plainColumn, final String encryptorName) {
+        this(logicColumn, null, cipherColumn, null, assistedQueryColumn, null, plainColumn, null, encryptorName);
+    }
+    
+    public EncryptColumnRuleConfiguration(final String logicColumn, final String logicDataType, final String cipherColumn, final String cipherDataType, final String assistedQueryColumn, 
+                                          final String assistedQueryDataType, final String plainColumn, final String plainDataType, final String encryptorName) {
+        this.logicColumn = logicColumn;
+        this.logicDataType = logicDataType;
+        this.cipherColumn = cipherColumn;
+        this.cipherDataType = cipherDataType;
+        this.assistedQueryColumn = assistedQueryColumn;
+        this.assistedQueryDataType = assistedQueryDataType;
+        this.plainColumn = plainColumn;
+        this.plainDataType = plainDataType;
+        this.encryptorName = encryptorName;
+    }
```

Comment:
Why not use @RequiredArgsConstructor here?

Reply:
Yes, I'll replace the construct with @RequiredArgsConstructor.

Reply:
@terrymanu Hi, I have replace all args construct with @RequiredArgsConstructor. Please take a look when you are free.